### DISCUSSION TITLE
[Question] How to properly add assets when running flet pack?

### DISCUSSION THREAD

**Discussion by wallsified** [flet-dev/flet#1917] [Q&A]
### Question

Hi. I'm trying to make an executable file for a proyect I made but I dont know what could be the proper way of adding the assets to the file. 

I currently have my project structure like [this: ](https://postimg.cc/7fTZdt0r). Hence, I dont really know what would be the adequate way of adding everything else. 

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
You can use the `--add-data` option.
